### DISCUSSION TITLE
Weblate plugin to translate things from mattermost

### DISCUSSION THREAD

**Discussion by jespino** [mattermost/community#135] [Enhancements]
Add a plugin that ask you time to time if you want to translate to your language, and then, whenever you say "yes", it start sending to you untranslated strings, and you have reply with the translations strings, and automatically get translate in weblate.

That can be optionally moderated, so somebody (in a list of people) get asked if he/she wants to review strings, and then the bot start showing review-pending strings, and only have to click on "accept", "reject", or "pass", and that way all gets reviewed from mattermost.

**Comment by chetanyakan** [comment]
OR you could have a channel of french translators.
Then, a bot would post a string to be translated to french in that channel and members of that channel can discuss amongst themselves and reply to the bot to send the translated string to Weblate.
